presets.txt

```diff
@@ -0,0 +1,3 @@
+default 
+pretty -fe 8000 -d 100 -ht 1080
+test -fr 13 -ht 418 -va -fs 520
```

Comment:
Should be changed, maybe filled with several presets we distribute as some sort of 'recommendation'.

Reply:
Will be done at a later date when and how we see fit